cmd/zpool/zpool_main.c

```diff
@@ -3248,51 +3371,54 @@ zpool_do_checkpoint(int argc, char **argv)
 /*
  * zpool import [-d dir] [-D]
  *       import [-o mntopts] [-o prop=value] ... [-R root] [-D] [-l]
- *              [-d dir | -c cachefile] [-f] -a
+ *              [-d dir | -c cachefile | -s] [-f] -a
```

Comment:
FWIW, I feel like having this manpage-like comment is not that useful since it has such an opportunity to get out of date with the "real" documentation in the manpage and help message.  And it adds a 3rd place that needs to be updated when changing the documentation.  So I'd be OK with removing it entirely if you want (rather than fixing it up as you've done here).